microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -197,49 +218,55 @@ The following steps demonstrate that edits aren't preserved across page refreshe
 
-1. Replace `<h1>Workspaces Demo</h1>` with `<h1>I Love Cake</h1>`.  The demo title changes.
+1. Replace `<h1>DevTools Workspaces Demo</h1>` with `<h1>I Love Cake</h1>`, in the file listing (as opposed to the DOM tree in the **Elements** tool).
 
 1. Press **Ctrl+S** (Windows, Linux) or **Command+S** (macOS) to save the change.
 
-1. Refresh the page.  The `<h1>` element continues to display the new text after the page is refreshed.
+1. Refresh the page.  The heading in the rendered page changes to "I Love Cake", because that string was saved in `index.html` in your mapped `app` directory:
 
    ![Change HTML from the Sources tool](./index-images/workspaces-workspaces-demo-sources-page-h1.png)
 
-1. Open `~/Desktop/app/index.html` in a text editor.  The `<h1>` element contains the new text.
+1. Open the copy of `index.html` that's in your `app` directory in a text editor, such as Visual Studio Code.
+
+   The `<h1>` element contains the new text, because you made the change using the **Sources** tool's editor to edit `index.html` and then saved the change, and that file was mapped in a workspace (the **Filesystem** tab), indicated by a green dot on the file's icon.
 
 
 <!-- ====================================================================== -->
 ## Edit JavaScript and save changes to the source file
 
 The main place to use the code editor of DevTools is the **Sources** tool.  But sometimes you need to access other tools, such as the **Elements** tool or the **Console**, while editing files.  The **Quick source** tool gives you just the editor from the **Sources** tool, while any tool is open.
+<!-- todo: avoid Quick source tool and just use Sources tool for this section -->
 
 To open the DevTools code editor alongside other tools:
 
-1. Select the **Elements** tool.
+1. Continuing from above, select the **Elements** (![Elements tool icon.](./index-images/elements-tool-icon.png)) tool.
 
 1. Press **Ctrl+Shift+P** (Windows, Linux) or **Command+Shift+P** (macOS) to open the **Command Menu**.
 
-1. Type **quick** at the **Run** prompt, and then select **Show Quick source**.  At the bottom of the DevTools window, the **Quick source** tool appears, displaying the contents of `index.html`, which is the last file you edited in the **Sources** tool.
+1. At the **Run** prompt, start typing **quick**, and then select **Show Quick source**:
 
    ![Open the 'Quick source' tool by using the Command Menu](./index-images/workspaces-workspaces-demo-search-show-quick-source.png)
+   <!-- todo: resume here -->
 
-1. Press **Ctrl+P** (Windows, Linux) or **Command+P** (macOS) to open the **Open File** dialog, shown in the next screenshot.
+   At the bottom of the DevTools window, the **Quick source** tool opens, displaying the contents of `index.html`, because that's the last file you edited in the **Sources** tool.  If needed, click **Expand Quick View**, and make sure the **Elements** tool is selected in the main toolbar.
```

Comment:
Shouldn't "main toolbar" be "Activity Bar" here?

Reply:
and make sure the **Elements** tool is selected.